### PR TITLE
Update with some changes

### PR DESCRIPTION
**Pull request by Croden1999**
- Add ch (Traditional Chinese) support.
- Update zh (Simplified Chinese) translation.
- Tiny format fixed.

### CODE REVIEW

File: _includes/languageselection.html [hunk 1]
```diff
@@ -45,10 +45,13 @@ <h5 class="modal-title" id="exampleModalLabel">{{texts[25]}}</h5>
...
                 <br>
                 <a href="#" data-dismiss="modal" ng-click="changeLang(12)"><span class="flag-icon flag-icon-cn"></span>
-                    中文</a>
+                    简体中文</a>
                 <br>
                 <a href="#" data-dismiss="modal" ng-click="changeLang(13)"><span class="flag-icon flag-icon-jp"></span>
                     日本語</a>
+                <br>
+                <a href="#" data-dismiss="modal" ng-click="changeLang(14)"><span class="flag-icon flag-icon-hk flag-icon-tw"></span>
+                    繁體中文</a>
```
Comment (nishinji) [inline _includes/languageselection.html:54]: Can you correct the alphabetical order to include Japanese and Chinese?

Comment (Croden1999) [reply _includes/languageselection.html:51]: Done, please check again

Comment (nishinji) [reply _includes/languageselection.html:51]: why two flags?
can you make it one?

Comment (Croden1999) [reply _includes/languageselection.html:51]: These both regions use Traditional Chinese as their official language, also I want add the Macau flag, but I not found it.

Comment (nishinji) [reply _includes/languageselection.html:51]: image
Well two flags won't appear 🤷‍♂️
Delete `flag-icon-hk`

File: _includes/nav.html [hunk 2]
```diff
@@ -63,7 +63,8 @@
                         <span ng-if="lang == 'pl'"><span class="flag-icon flag-icon-pl"></span> PL</span>
                         <span ng-if="lang == 'ru'"><span class="flag-icon flag-icon-ru"></span> RU</span>
                         <span ng-if="lang == 'vn'"><span class="flag-icon flag-icon-vn"></span> VN</span>
-                        <span ng-if="lang == 'zh'"><span class="flag-icon flag-icon-cn"></span> CN</span>
+                        <span ng-if="lang == 'zh'"><span class="flag-icon flag-icon-cn"></span> ZH</span>
+                        <span ng-if="lang == 'ch'"><span class="flag-icon flag-icon-hk flag-icon-tw"></span> CH</span>
```
Comment (nishinji) [inline _includes/nav.html:67]: same

**Comment by nishinji** [comment]
Nice 👍

**Comment by Croden1999** [comment]
> Nice 👍

Thx, sir.

**Pull request merged**